src/components/StatisticsDisplay.vue

```diff
@@ -57,13 +55,15 @@ export default class StatisticsDisplay extends Vue {
     const sortedStatistics = Object.entries(this.statistics).sort(
       (category, nextCategory) => nextCategory[1] - category[1]
     );
-
     const filteredStats = sortedStatistics.filter(item => parseInt(this.getCategoryPercentage(item[0])) > 5);
     const miscellaneousCategory = sortedStatistics
       .filter(item => parseInt(this.getCategoryPercentage(item[0])) <= 5)
       .reduce((item, prev) => ['miscellaneous', prev[1] + item[1]], ['miscellaneous', 0]);
 
-    this.statistics = Object.fromEntries([...filteredStats, miscellaneousCategory]);
+    const filteredStatsWithMiscCategory =
+      miscellaneousCategory[1] !== 0 ? [...filteredStats, miscellaneousCategory] : filteredStats;
+
+    return Object.fromEntries(filteredStatsWithMiscCategory);
```

Comment:
If we apply the filter for dropping categories with value of <5 at the end, then we can get rid of lines 63 and 64

Reply:
It would result in loosing miscellaneous category, which we always want to display